recipe/meta.yaml

```
@@ -10,34 +10,48 @@ source:
   sha256: fe859b0e98d222b4b458022007daac2757e970f0671d45e55d01931c1b4bb3cb
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv "
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cffi                                   # [build_platform != target_platform]
+    - pip
     - {{ compiler('c') }}  # [not (win and py27)]
   host:
-    - cffi >=1.0.0
-    - pip
+    - cffi =>1.15.0
     - python
+    - pip
+    - wheel
+    - m2w64-toolchain 5.3.0 # [win and py27]
+    - libpython 2.1       # [win and py27]
+    - {{ pin_compatible('m2w64-gcc-libs', max_pin='5.3.0') }}   # [win and py27]
+
   run:
-    - cffi >=1.0.0
+    - cffi =>1.15.0
     - python
+    - m2w64-gcc-libs   # [win and py27]
 
 test:
+  skip: True  # [osx]
```

```suggestion
  skip: True  # [osx]
```
Why are osx tests skipped?

---

This skip must be removed